docs/usage/configuration-options.md

```diff
@@ -3466,6 +3450,38 @@ This is why we configured an upper limit for how long we wait until creating a P
 !!! note
     If the option `minimumReleaseAge` is non-zero then Renovate disables the `prNotPendingHours` functionality.
 
+## prOptions
+
+Use `prOptions` to set pr-specific configuration options provided by Renovate.
+
+Example:
+
+```json
+{
+  "prOptions": {
+    "gitLabIgnoreApprovals": true
+  }
+}
+```
+
+You can use these options:
```

Comment:
```suggestion
You can use these configuration options within `prOptions`:

- `azureWorkItemId`
- `bbUseDefaultReviewers`
- `gitLabIgnoreApprovals`
```

Explain that we mean you can use these options within `prOptions`, and list the options.

Even better would be to autogenerate a table of allowed options within `prOptions`. That way, when we later add new options, the docs are updated automatically. I think we already do this for other bits of the docs.